CdCl_xfoil.m

```diff
@@ -0,0 +1,122 @@
+% MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+% General Public License for more details.
+% <http://www.gnu.org/licenses/>.
+%
+% ==============================================================================================
+% |Name        : CdCl_xfoil.m                                                                    |
+% |Author(s)   : Lorenzo Frascino - Palma Caputo                                                 |
+% |              University of Naples Federico II.                                               |
+% |Version     : 1.0                                                                             |
+% |Date        : 03/01/2020                                                                      |
+% |Modified    : 03/01/2020                                                                      |
+% |Description : evaluation of the Cd(Cl) polar by using the software xfoil                      |     
+% |Reference   : xfoil user's Guide                                                              |
+% |              <https://web.mit.edu/drela/Public/web/xfoil/xfoil_doc.txt>                      |                                                         
+% |Input       : (NACA) = NACA airfoil                                                           |
+% |              (ext_AF) = flag if airfoil geometry must be imported from external file         |  
+% |              (numPanel) = numver of panels for the geometry                                  |
+% |              (Re_number) = Reynolds number                                                   |
+% |              (iter) = number of iterations                                                   |
+% |              (FirstAlfa) = first value of the angle of attack                                |
+% |              (LastAlfa) = last value of the angle of attack                                  |
+% |              (DeltaAlfa) = pace between the angles of attack                                 |
+% |Output      : (Cl) = lift coefficient vector                                                  |
+% |              (Cd) = drag coefficent vector                                                   |
+% |Note        : The function must be in the same directory of xfoil.exe!                        |
+% ==============================================================================================
+
+Alfa_vec   = str2num(FirstAlfa):str2num(DeltaAlfa):str2num(LastAlfa);
+saveGeometry = 'Airfoil_geometry.txt';  % Create .txt file to save airfoil coordinates
+savePolar  = 'Polar.txt';               % Create .txt file to save the polar
+
+
+% Delete files if they exist
+if (exist(saveGeometry,'file'))
+    delete(saveGeometry);
+end
+
+if (exist(savePolar,'file'))
+    delete(savePolar);
+end
+
+
+%% WRITING XFOIL COMMANDS
+% Create the airfoil
+f_input = fopen('xfoil_input.txt','w');            % Create input file for xfoil 
+
+if ext_AF == 0
+        fprintf(f_input,'y\n');

```

Comment:
The indentation should be the same everywhere.
A good way to do it is to use 4 spaces.